examples/protocols/pppos_client/components/modem/src/bg96.c

```diff
@@ -467,8 +474,6 @@ modem_dce_t *bg96_init(modem_dte_t *dte)
     DCE_CHECK(bg96_get_imei_number(bg96_dce) == ESP_OK, "get imei failed", err_io);
     /* Get IMSI number */
     DCE_CHECK(bg96_get_imsi_number(bg96_dce) == ESP_OK, "get imsi failed", err_io);
-    /* Get operator name */
-    DCE_CHECK(bg96_get_operator_name(bg96_dce) == ESP_OK, "get operator name failed", err_io);
```

Comment:
Removing `get_operator_name()` from init might cause an inconsistency in idf examples. I would suggest adding
```
    ESP_ERROR_CHECK(dce->get_operator_name(dce));
```
to the `pppos_client_main.c` after the specific initialization section.

This may require applying the same changes to other devices (SIM800, ..) as well

Reply:
fallback